easybuild/easyconfigs/c/COSMO_pompa/COSMO_pompa-31d7227-CrayCCE-18.08-cordex-double.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'MakeCp'
+
+name = 'COSMO_pompa'
+version = '31d7227'
+implementation = 'cordex'
+precision = 'double'
+versionsuffix = '-%s-%s' % (implementation, precision)
+
+homepage = 'https://github.com/C2SM-RCM/cosmo-pompa/tree/master/dycore (-b crclim)'
+description = """
+cosmo-pompa
+"""
+
+toolchain = {'name': 'CrayCCE', 'version': '18.08'}
+toolchainopts = {'verbose': False}
+# repo is private (+ no official release)
+sources = [
+    {'filename': '/apps/common/UES/easybuild/sources/c/COSMO_pompa_crclim/'    \
+        'cosmo_pompa_crclim_31d7227a03e90386fa9f7884e0c7bed4e2ae2a81.tar.gz'}
+]
+
+builddependencies = [
+    ('CMake', '3.12.0', '', True), 
+    ('libgrib1_crclim', 'a1e4271'),
+    ('Dycore/551f6b4-CrayGNU-18.08-%s-%s' 
+      % ( implementation, precision), EXTERNAL_MODULE),
+]
+
+dependencies = [ 
+    ('cray-netcdf/4.6.1.2', EXTERNAL_MODULE),
```

Comment:
Why `cray-netcdf` does not hurt here?

Reply:
Cray-netcdf here works, because I export certain environment variables anew. Before, EB set cray-netcdf for GNU and never changed it at a later point in time.
(the two files, which still had cray-netcdf outside the dependencies had been committed earlier and I oversaw the, when I removed the rest. Sorry for that...)